app/javascripts/components/makeGame/fourthStep/index.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { push } from 'react-router-redux';
+import Icon from '../../../icons';
+import SolidButton from '../../common/solidButton';
+import EmptySolidButton from '../../common/emptySolidButton';
+// actions
+import { setBetMinValue, setBetMaxValue } from '../actions';
+// styles
+import styles from './fourthStep.scss';
+
+function mapStateToProps(appState) {
+  return {
+    rootState: appState.root,
+    makeGameState: appState.makeGame,
+  };
+}
+
+class MakeGameFourthStep extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.validateValidValues = this.validateValidValues.bind(this);
+  }
+
+  render() {
+    const { makeGameState } = this.props;
+
+    let maxBet = 0;
+    if (makeGameState.get('maxPrize') && makeGameState.get('totalStake')) {
+      maxBet = parseFloat(makeGameState.get('totalStake'), 10) / parseFloat(makeGameState.get('maxPrize'), 10);
```

Comment:
This is for the safe calculation. because either totalStake and maxPrize can be string type from anywhere.